lib/brakeman/checks/base_check.rb

```diff
@@ -280,15 +280,6 @@ def get_location result
     return location, line
   end
 
-  #Checks if an expression contains string interpolation.
-  #
-  #Returns Match with :interp type if found.
-  def include_interp? exp
```

Comment:
This was only used in `check_execute.rb` so I moved it there.

Reply:
Okay. Might move back at some point :)